### PR TITLE
feat: Add 11123 and 6785, updating metadata

### LINKED ISSUE #92: Reprint Canon Titles 52 (correct numbering)
```
52	Johnstone	A Sunless Heart	1894 6785__johnstone_sunless_heart_chp17of24_seed6785.html 
```
broadview

### LINKED ISSUE #94: Reprint Canon Titles 54 (correct numbering)
```
11123__harkness_city_girl_chp1of13_seed11123.html
```
broadview

### PR DESCRIPTION
**Pull request by riddella**
Closes #92
Closes #94

### CODE REVIEW

**Comment by riddella** [comment]
@ayarnell spot anything amiss?

File: texts/6785__johnstone_sunless_heart_chp17of24_seed6785.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+<p>“As white,” said Gasparine steadily, “as snow—when it fell, trampled perhaps by busy, thoughtless, careless feet ... but white, to my eyes, Lo, always.”</p>
+<p>The great tears stood in her eyes.</p>
+<p>“Ah! Then you had better know.... Why should I hide it, after that? I see you are the sort of girl will worship me the more for it.... Then, listen ...”</p>
+<p>Lo, too, seemed moved.</p>
+<p>Still, Gasparine saw the next words came with a frightful effort, for the white face changed from pallid white to pallid green. At last, contemptuously, the words came.</p>
+<p>“One hears so often of wrongs—so often of that ... great wrong.”</p>
+<p>Suddenly she bent towards Gasparine. She spoke rapidly and low, her breath came quick, her eyes were terrible.</p>
+<p>“A man once ...”</p>
+<p>Over Gasparine’s face came a look of indescribable horror. Sometimes, as she listened, she gave a cry.</p>
+<p>*<span class="space">*</span> <span class="space">*</span> <span class="space">*</span> <span class="space">*</span></p>
```
Comment (ayarnell) [inline texts/6785__johnstone_sunless_heart_chp17of24_seed6785.html]: @riddella Are these `span` tags meant to be here, or are they a Project Gutenberg artifact?

**Comment by ayarnell** [comment]
@riddella Some span tags in 6785 that I think are not meant to be there. Everything else looks good!

File: texts/6785__johnstone_sunless_heart_chp17of24_seed6785.html (hunk 1 above)
Comment (riddella) [reply texts/6785__johnstone_sunless_heart_chp17of24_seed6785.html]: Good catch.

**Pull request merged**